config/intel-warnings/general

```diff
@@ -1,2 +1 @@
 -Wall
--Wcheck
```

Comment:
Checking for the suitability of this flag should be done in config/intel-flags, etc. instead of being removed for everyone.